### PR TITLE
chore: add default issue templates for the github org

### PR DESCRIPTION
**Pull request by jooola**
Add forms for bug report or feature request, for repositories that do not implement their own.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (apricote) [inline .github/ISSUE_TEMPLATE/config.yaml:1]: Are you sure you want to disable blank issues? There are some types of requests that do not match "Bug" or "Feature". E.g. someone could have questions about how a feature can be used (https://github.com/hetznercloud/hcloud-cloud-controller-manager/issues/517)

**Review by apricote — APPROVED** [review]
Looking good besides the blank issues :)

File: .github/ISSUE_TEMPLATE/config.yaml (hunk 1 above)
Comment (jooola) [reply .github/ISSUE_TEMPLATE/config.yaml]: Note that these templates will only be used by projects that don't have **any** Issue templates configured. If a single file is present, none of the defaults will be shown.

**Comment by jooola** [comment]
Note that these templates will only be used by projects that don't have **any** Issue templates configured. If a single file is present, none of the defaults will be shown.

**Pull request merged**